src/aap_eda/tasks/orchestrator.py

```diff
@@ -294,13 +294,13 @@ def dispatch(
                     ActivationStatus.WORKERS_OFFLINE,
                     msg,
                 )
-                LOGGER.warning(msg)
+                LOGGER.error(msg)
                 return
 
             # The queue is unhealthy, but this is a restart.
             # The priority is to adhere to the restart policy and
             # execute the task.
-            LOGGER.warning(
+            LOGGER.error(
```

Comment:
I would say here warning is appropriate, it is a legal restart after workers-offline.

Reply:
Agree, I would say it has an excess of info. 
What about "                f"Forcing user restart of {process_parent_type} {process_parent_id} "
                "on the least busy queue; after failing liveness checks of current associated queue."